doozer/tests/test_metadata.py

```diff
@@ -72,7 +72,7 @@ def test_is_rpm_exempt(self):
 
     def build_record(self, creation_dt: datetime.datetime, assembly, name='foo-container',
                      version='4.7.0', p='p0', epoch=None, git_commit='4c0ed6d',
-                     release_prefix=None, release_suffix='',
+                     release_prefix=None, el_target=8,
```

Comment:
Suggesting that el_target be a required parameter. The default make it harder to reason about each individual test result without looking at the implementation of `build_record`.

Reply:
done